cob_moveit_bringup/launch/move_group.launch

```diff
@@ -54,7 +56,8 @@
     <param name="allow_trajectory_execution" value="$(arg allow_trajectory_execution)"/>
     <param name="max_safe_path_cost" value="$(arg max_safe_path_cost)"/>
     <param name="jiggle_fraction" value="$(arg jiggle_fraction)" />
-
+    <param name="collision_detector" value="IndustrialFCL" />
```

Comment:
it's defined in https://github.com/ros-industrial/industrial_moveit/blob/indigo-devel/industrial_collision_detection/industrial_collision_detection_plugin_description.xml
so we need a `<exec_depend>industrial_collision_detection</exec_depend>`